src/docs/sdk/client-reports.mdx

```diff
@@ -83,6 +83,9 @@ The following discard reasons are currently defined for `discarded_events`:
 - `before_send`: an event was dropped in `before_send`
 - `event_processor`: an event was dropped by an event processor
 
+In case a reason needs to be added,
+it also has to be added to the allowlist in snuba (`outcomes_processor.py`).
```

Comment:
I'd rather point to a repo permlink.

Reply:
here you go: https://github.com/getsentry/develop/pull/549